src/libsyntax/feature_gate.rs

```diff
@@ -443,6 +446,19 @@ pub fn deprecated_attributes() -> Vec<&'static (&'static str, AttributeType, Att
     BUILTIN_ATTRIBUTES.iter().filter(|a| a.2.is_deprecated()).collect()
 }
 
+pub fn is_builtin_attr<A>(attr: &A) -> bool where A: PartialEq<&'static str> {
```

Comment:
Its only use currently is with `Symbol`, but other code that wants to check this later might use other types.